lib/forms/field-helpers.js

```diff
@@ -207,3 +208,67 @@ export function getValidationError(data, validate, store, name) { // eslint-disa
     return messages.max;
   }
 }
+
+/**
+ * determine if something should be revealed based on the sites specified
+ * and the current site
+ * @param  {string} sites
+ * @param  {string} currentSlug
+ * @return {boolean}
+ */
+function shouldBeRevealedBySite(sites, currentSlug) {
+  return !!filterBySite([{ sites }], currentSlug).length;
```

Comment:
this might be a future feature request, but it might be useful to have this filterable by more than just the site (e.g. i had a need for nymag site, but subscription pages only: `nymag.com/magazine/subscription`)

Reply:
Ah, that might be useful. I think we'd have to do some kind of discovery though, since I'm not sure what the scope of use cases would be.